### DISCUSSION TITLE
Active/active replication

### DISCUSSION THREAD

**Discussion by benesch** [MaterializeInc/materialize#6971] [Technical musings]
To start, Materialized will require a hardcoded list of nodes in the cluster. The future API should perform cluster bootstrapping at the Materialized level, but not start any Timely clusters until the user acknowledges that the node membership is reasonably fixed (so we don't waste time tearing down/spinning up clusters for every node as nodes join).

**Comment by rjnn** [comment]
Dup #291. Close this and continue there?

**Comment by benesch** [comment]
I think #291 was the duplicate. 😄

**Comment by rjnn** [comment]
Can an active-active cluster cross availability zones?

**Comment by cuongdo** [comment]
Whether or not we can span AZs probably depends on the nature of the communications between the two actives. If they're chatty with one another, maybe that's not great, because that'll rack up bandwidth costs quickly. We don't yet know enough to answer this question, but we should start progressing on active/active soon.
